tests/test_load_results/test_mteb_load_results.py

```diff
@@ -23,3 +46,230 @@ def test_mteb_load_results():
     known_revision = "6d9c09a789ad5dd126b476323fccfeeafcd90509"
     assert known_model in results
     assert known_revision in results[known_model]
+
+
+@pytest.mark.xfail
+@pytest.mark.parametrize("task", MTEB().tasks_cls)
+def test_load_results_main_score_in_real_results(task):
```

Comment:
Yes, but when I tried to `create_meta` for models that run before some changes in `STS` create meta just failed, because it needed to have task main score in all scores. This was the problem that I tried to fix with this PR. https://github.com/embeddings-benchmark/mteb/blob/main/mteb/cli.py#L272

Reply:
Ahh right. The new fix to STS should probably fix that. This seems like a test of create_meta though rather than of the results format